_dbt/models/marts/appointment/fact_appointment.sql

```diff
@@ -0,0 +1,46 @@
+with appointment_data as(
+    select  Appointment_id,
+            Patient_id,
+            Doctor_id,
+            Appointment_date
+    from {{ref ('stg_appointment')}}              
+),
+patient_data as (
+    select 
+        Patient_id,
+        email
+    from {{ref ('dim_patient')}}
+),
+doctor_data as (
+    select Doctor_id
+    from {{ref ('dim_doctor')}}
+),
+
+procedure_data as (
+    select  Procedure_id,
+            Appointment_id
+           
+    from {{ref('dim_medical_procedure')}}
+),
+billing_data as (
+    select  Invoice_id,
+            Patient_id 
+    from {{ref ('dim_billing')}}
+)
+
+select
```

Comment:
It would be helpful to have either group by or distinct to ensure uniqueness for the rows